### ISSUE TITLE
Level 70 Boost

### ISSUE THREAD

**Issue by jamiec18** [dalaranwow/dalaran-wow#5803]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who): Logging into Website, no option for free boost despite many articles telling me 'm eligible, really want to start playing on the server and have waited for this boost for a while now.

**Expected behaviour**: I should be made eligible for a free boost on your websiote, or you boost my in game character (scryerwaitno, draenei Shaman).

**Steps to reproduce the problem**:

1.  Check character services, boost 
2. Check price, select heirlooms, try check out

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Promotion:  ![Ss 1 DW](https://user-images.githubusercontent.com/53865620/62625402-045e7480-b91d-11e9-932e-74f7274d8f8e.PNG)

Date: 
![Ss 2 DW](https://user-images.githubusercontent.com/53865620/62625410-0a545580-b91d-11e9-8f18-98cb195f60f9.PNG)

Tournament hasn't started yet, so surely I'm eligible?

**Comment by Golganeth** [comment]
Greetings!
The bug-tracker only lists in-game issues so this should go to our forums.

We initially thought to keep the promotion until the tournament started, but since the Tournament will start shortly, we considered we had given enough time to players to get the boost and get to Level 80. Our only purpose offering this boost was to give more players a chance to participate, and players who would get it at this point would (probably) not be part of it.

You can read this [article](https://www.dalaran-wow.com/news/146/quick-notes/) where we inform about it:

> Additionally, we would like to communicate that the following promotions will NOT be active anymore from now on:
>
> - Free instant 70 promotion (we won't be accepting new applications either).
> - Free Faction Change from Horde to Alliance.

The article is from 4th of July. The current date is 7th August indeed!

![image](https://user-images.githubusercontent.com/5057942/62630329-03ced980-b92f-11e9-8c78-8cd09c06ef91.png)

Closing up.

**Comment by Chenocide** [comment]
There's currently x2 experience, lasting until 19th of August.

**Comment by Golganeth** [comment]
Hello,
Yes, currently we have Double XP, Refer a Friend until Level 80, and gifts for players that were already playing with us before our 6th Anniversary! 
The whole promotion will be active until August 19th. 
More info can be [read](https://www.dalaran-wow.com/news/148/6th-anniversary-gifts-double-xp-and-raf-until-level-80/) here.
